src/create_disk.sh

```diff
@@ -392,7 +395,7 @@ s390x)
 	# stage on s390x, either through zipl->grub2-emu or zipl standalone.
 	# See https://github.com/coreos/ignition-dracut/issues/84
 	# A similar hack is present in https://github.com/coreos/coreos-assembler/blob/master/src/gf-platformid#L55
-	echo "$(grep options $blsfile) ignition.firstboot rd.neednet=1 ip=dhcp,dhcp6" > $tmpfile
```

Comment:
Hmm actually, I think we need to add `${firstboot_kargs}` here, right? Because s390x doesn't use the GRUB config yet. And in `gf_platformid` too.

Or... we just sprint to https://github.com/coreos/fedora-coreos-config/pull/426.

Reply:
This will also affect RHCOS and slower FCOS streams, so let's fix this.

I'm unsure however how to wire this new variable content to `gf_platformid`. Do we actually need to have it there, if that is only injecting the platform ID?

Reply:
Meh, my current attempt is at https://github.com/coreos/coreos-assembler/pull/1479.